### DISCUSSION TITLE
fluidsynth 2.2 Beta1 pre-built binary missing libraries?

### DISCUSSION THREAD

**Discussion by jimhen3ry** [FluidSynth/fluidsynth#798] [Q&A] [answered]
The Windows 10 x64 pre-built binary for 2.2 Beta1 is missing libgcc_s_seh-1.dll, libgomp-1.dll, and livstdc++-6.dll. Is this intentional? If so, what is the preferred way of obtaining the missing libraries?

**Comment by derselbst** [comment, marked as answer]
Nope, that's a mistake. They should have been included among all the other dlls. I'll fix that for the next release.

**Reply by jimhen3ry** [reply]
Thanks!
